testcases/PowerNVDump.py

```diff
@@ -1068,6 +1068,72 @@ def runTest(self):
             else:
                 log.info("PASSED: Hugepage size is {} kB".format(hugepage_size))
 
+class KernelCrash_XIVE_off(PowerNVDump):
+    '''
+    This test checks kdump/fadump with kernel parameter option xive=off 
+    '''
+
+    def runTest(self):
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.setup_test()
+        log.info("=============== Testing kdump/fadump with xive=off ===============")
+        mmu = self.c.run_command("awk '$1 == \"MMU\" {print $3}' /proc/cpuinfo")[0]
+        log.debug(" MMU '{}'".format(mmu))
+        if mmu == "Radix":
+            self.cv_SYSTEM.goto_state(OpSystemState.OS)
+            obj = OpTestInstallUtil.InstallUtil()
+            if not obj.update_kernel_cmdline(self.distro, args="xive=off",
+                                             reboot=True, reboot_cmd=True):
+                self.fail("KernelArgTest failed to update kernel args")
+            self.cv_SYSTEM.goto_state(OpSystemState.OFF)
+            self.cv_SYSTEM.goto_state(OpSystemState.OS)
+            boot_type = self.kernel_crash()
+            self.verify_dump_file(boot_type)
+            kernel_boottime_arg = self.c.run_command("cat /proc/cmdline | grep -o 'xive=off'")[0]
```

Comment:
Can you move this check before triggering the crash? Incase the kernel command line setup fails we should not trigger a crash.

Reply:
Done